react-time-track/src/components/add-member-project.js

```diff
@@ -5,8 +5,137 @@ import { FaAngleDown, FaAngleUp } from "react-icons/fa";
 
 import { Button } from "./ui";
 
+const sectionStyle = {
```

Comment:
We have to move all those style elements somewhere else. Or, may be, divide this component into the necessary views and executing ones.

Reply:
Not required to do right now. =|